package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.0.0-development",
   "description": "NPM Library for Generating Static Cucumber Reports",
   "main": "src/Generator.js",
+  "types": "types/Generator.d.ts",
   "scripts": {
     "test": "./node_modules/.bin/cucumber-js features/*.feature",
-    "lint": "npx eslint src --color"
+    "lint": "npx eslint src --color",
+    "build:types": "npx -p typescript tsc -p tsconfig.declaration.json",
+    "prepublishOnly": "npm run build:types"
```

Comment:
Out of curiosity, (since I am totally ignorant of TS) what does this do?  And why would we only run it during the prepublish?

Reply:
ah, it's so that in case they haven't been generated, and so that we don't have to do it manually, they'll be included on the released artifacts